tests/TestApp/AdalDesktopTestApp/Program.cs

```diff
@@ -54,11 +59,66 @@ static void Main(string[] args)
 
         private static async Task AcquireTokenAsync()
         {
-            AuthenticationContext context = new AuthenticationContext("https://login.microsoftonline.com/common", true);
-            var result = await context.AcquireTokenAsync("https://graph.windows.net", "<CLIENT_ID>", new UserCredential("<user>"));
+            Telemetry telemetry = Telemetry.GetInstance();
+            DispatcherImplement dispatcher = new DispatcherImplement();
+            telemetry.RegisterDispatcher(dispatcher, false);
+
+            //LoggerCallbackHandler.Callback = new MyCallback();
 
+            AuthenticationContext context = new AuthenticationContext("https://login.microsoftonline.com/common", true);
+            /*var result =
+                await
+                    context.AcquireTokenAsync("https://graph.windows.net", "3b5d8539-60f9-4158-8d3f-c3b9cae4a149",
+                        new Uri("urn:ietf:wg:oauth:2.0:oob"), new PlatformParameters(PromptBehavior.Auto),
+                        new UserIdentifier("f9fd32ba-4c8c-43ca-a62c-09ab16f1dd3e", UserIdentifierType.UniqueId), "prompt=consent");*/
+            var result = await context.AcquireTokenAsync("https://graph.windows.net", "193faa18-0c0b-45f3-9125-b08ff04d9890", new UserPasswordCredential("test@abgun.onmicrosoft.com", "P@ssword<"));
+            //var result = await context.AcquireTokenAsync("https://graph.windows.net", "193faa18-0c0b-45f3-9125-b08ff04d9890", new Uri("urn:ietf:wg:oauth:2.0:oob"), new PlatformParameters(PromptBehavior.Always));
+            TokenCache.DefaultShared.Clear();
             string token = result.AccessToken;
             Console.WriteLine(token + "\n");
+            dispatcher.file();
```

Comment:
>dispatcher.file(); [](start = 12, length = 18)

this is an incorrect example of the telemetry event dispatching. you should not need to call anything as an app developer. Please update the sample to reflect the correct usage of the Telemetry feature.